ovos_plugin_manager/templates/tts.py

```diff
@@ -246,7 +236,11 @@ def handle_metric(self, metadata=None):
     def load_spellings(self, config=None):
         """Load phonetic spellings of words as dictionary."""
         path = join('text', self.lang.lower(), 'phonetic_spellings.txt')
-        spellings_file = resolve_resource_file(path, config=config)
+        try:
+            spellings_file = resolve_resource_file(path, config=config)
+        except:
+            LOG.exception('Failed to locate phonetic spellings resouce file.')
```

Comment:
this file is optional, can you make the log level lower? maybe debug. It will be common for this to be missing and thats fine